pages/index.vue

```diff
@@ -0,0 +1,519 @@
+                    di <b><em class="text-white">appassionati, sportivi</em></b>, <br />
+                    <b><em class=" text-white">aziende del settore</em></b> e investitori <br />
+                    <b><em class="text-white">interessati alle innovazioni</em></b> <br>
+                    web3. 
+                </div>
+                <img src="/partner.png" alt=""
+                    class="absolute -right-10 top-1/2 transform -translate-y-1/2 hidden lg:inline " />
+            </div>
+
+        </div>
+    </section>
+
+    <!-- new era section -->
+    <section class="mt-24 md:mt-[203px]">
+        <div class="container">
+            <div class="text-white">
+                <h4 class="text-4xl md:text-5xl lg:text-6xl">PadelDAO: <br>
+                    una nuova Era per il tuo Padel Club
+                </h4>
+            </div>
+            <div class="border-[0.5px] border-white/50 mt-20 divide-y divide-white/50">
+                <!-- top -->
+                <div class="flex flex-col lg:flex-row divide-y lg:divide-y-0 lg:divide-x divide-white/50">
+                    <div class=" basis-7/12 bg-[url('/galaxy.png')] bg-no-repeat bg-right bg-cover sm:bg-contain">
+                        <div class="px-8 pb-4 lg:pb-0 space-y-2.5 mt-10">
+                            <p class="text-4xl lg:text-5xl text-white font-medium">
+                                Aumento visibilità <br>
+                                & accessi
+                            </p>
+                            <p class=" text-base text-[#99A9AB]">
+                                Offre visibilità all’interno dei canali social,<br>
+                                del sito web e più in generale all'interno <br>
+                                della nostra community.
+                            </p>
+                        </div>
+                    </div>
+                    <div class=" basis-6/12 divide-y divide-white/50">
+                        <div
+                            class=" py-4 px-5 space-y-2.5 bg-[linear-gradient(88deg,#00282E_47%,rgba(0,40,46,0)_100%),url('/study-play.png')] bg-no-repeat bg-right bg-cover md:bg-contain">
+                            <p class=" text-2xl text-white font-semibold">Study & Play</p>
+                            <p class=" text-[#99A9AB] text-base">Offre un'esperienza all-inclusive, comprendente <br>
+                                avanzamento sportivo, soggiorni di alta
+                                qualità, <br> gastronomia e intrattenimento, sottolineando i <br> vantaggi esclusivi per
+                                i
+                                nostri
+                                partner.</p>
+                        </div>
+                        <div
+                            class=" py-4 px-5 space-y-2.5  bg-[linear-gradient(88deg,#00282E_47%,rgba(0,40,46,0)_100%),url('/clinic-journey.png')] bg-no-repeat bg-right bg-cover md:bg-contain">
```

Comment:
all this: `bg-[linear-gradient(88deg,#00282E_47%,rgba(0,40,46,0)_100%),url('/clinic-journey.png')]` is making all the html difficult to read, on such of customization please create a custom class in the style section of the page for example:
.customBg{
// here the equivalent of: bg-[linear-gradient(88deg,#00282E_47%,rgba(0,40,46,0)_100%),url('/clinic-journey.png')]
}